### ISSUE TITLE
Version 2.8.x is missing

### ISSUE THREAD

**Issue by patrick91** [pre-commit/mirrors-prettier#29]
This might be because there's a v3 alpha 

https://github.com/prettier/prettier/commit/dcc0623911f8b352eda18f60683eb1a232d6c2bd

**Comment by gaborbernat** [comment]
But what if someone wants to use the non-alpha version? Should it still not backlink to an earlier version?

**Comment by asottile** [comment]
if you can think of a way to make the mirroring system work with that then I'm all ears, it doesn't sound easy and mirrors are meant to be a stopgap and not a solution

**Comment by patrick91** [comment]
@asottile the solution would be prettier (in this case) having an official pre-commit hook, right?

**Comment by asottile** [comment]
possibly, they used to but it was always broken which is why I begrudgingly made the mirror

**Comment by Tenzer** [comment]
I have taken a stab at fixing pre-commit-mirror-maker so it shouldn't skip releases like this: https://github.com/pre-commit/pre-commit-mirror-maker/pull/157.

**Comment by antonagestam** [comment]
@asottile If this needs to be broken, doesn't it make sense for it to be broken in the _other_ way? I.e. to not mirror alpha releases at all?

**Comment by asottile** [comment]
no?

**Comment by antonagestam** [comment]
Sigh, I guess it's completely possible to create a separate mirror that behaves that way instead.

Would you review a PR that adds an explanation of this behavior to the README?

**Comment by asottile** [comment]
I'll review anything, doesn't mean I'll merge it though

**Comment by antonagestam** [comment]
@asottile Sorry for being vague, I'm asking beforehand to get an idea of whether or not you think it's worth spending time on.

**Comment by asottile** [comment]
I mean you'd probably do better to spend time to think about a better mirroring strategy that keeps all the versions -- I just don't care enough about the problem since all of the things I use only go forward

**Comment by antonagestam** [comment]
Right, I'm not saying that I won't, really just offering to improve things in a small way. I can only give you feedback from my own experience as a user of this mirror: I would have appreciated knowing up-front that it doesn't mirror all stable release tags, that's why I thought it makes sense to clearly communicate that in the README. Since you are not saying that you think it is worthwhile, I won't spend the time.

**Comment by LukeNotable** [comment]
As a workaround I've been doing this, though not sure how good or bad it may be:

```
  - repo: https://github.com/pre-commit/mirrors-prettier
    rev: "v2.7.1"
    hooks:
      - id: prettier
        files: '\.(jsx?|tsx?|css)$'
        additional_dependencies:
          - prettier@2.8.0
```

**Comment by fregante** [comment]
I think this should be fixed.

- Alpha releases should not be used in production and thus not here either
- Adding support for releases is as easy as adding a secondary branch and manually creating commits and tags to it (not ideal, but not super involved either)
- Prettier v3 is months away, no progress was made in the last 2 months in the milestone tracker: https://github.com/prettier/prettier/milestone/14

**Comment by fregante** [comment]
Here's a solution for anyone landing here:

1. Fork the repo
2. Change 2 files
3. Create tag
4. Update your pre-commit config to point to your repo

Or just use my config:

```yml
  - repo: https://github.com/fregante/mirrors-prettier
    rev: v2.8.1
    hooks:
      - id: prettier
```

**Comment by asottile** [comment]
@fregante please actually read the conversation and relevant discussions first

**Comment by fregante** [comment]
You're actively hurting your own project. I supplied a perfectly valid solution and you hid the comment. Good job.

**Comment by asottile** [comment]
there's a better solution right above yours that's way less work

**Comment by fregante** [comment]
That's not better at all since it installs two versions

**Comment by asottile** [comment]
it doesn't install two versions
